charts/opentelemetry-collector/templates/clusterrole.yaml

```diff
@@ -42,4 +42,9 @@ rules:
     resources: ["nodes/stats"]
     verbs: ["get", "watch", "list"]
   {{- end }}
+  {{- if .Values.presets.kubernetesObjects.enabled }}
```

Comment:
I think you will need to add this into the overall if statement on line 1. You will also need to add it to the first line of the cluster role binding.

I've addressed a similar issue here: https://github.com/open-telemetry/opentelemetry-helm-charts/pull/687

Reply:
Oh cool!
Got it and fixed thanks @cjbhaines

Reply:
LGTM!